client/src/pages/OnBoard.js

```diff
@@ -0,0 +1,223 @@
+import React, { useState } from "react";
+//Material-ui imports
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import Container from "@material-ui/core/Container";
+import Background from "../elements/Background";
+import IconButton from "@material-ui/core/IconButton";
+import RemoveCircleIcon from "@material-ui/icons/RemoveCircle";
+import AddCircleIcon from "@material-ui/icons/AddCircle";
+import MenuItem from "@material-ui/core/MenuItem";
+import TextField from "@material-ui/core/TextField";
+import SubmitButton from "../elements/SubmitButton";
+import StyledPaper from "../elements/StyledPaper";
+import PageHeader from "../elements/PageHeader";
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    width: "100%",
+    marginTop: theme.spacing(3),
+    textAlign: "center",
+  },
+  add: {
+    margin: theme.spacing(3, 0, 2),
+    padding: theme.spacing(1, 6),
+    border: 0,
+    borderRadius: 25,
+    textTransform: "none",
+    fontSize: 16,
+  },
+  removeIcon: {
+    color: theme.palette.button.remove,
+  },
+}));
+
+export default function OnBoard() {
+  const classes = useStyles();
+
+  const languages = [
+    "Python",
+    "JavaScript",
+    "Java",
+    "PHP",
+    "GoLang",
+    "C#",
+    "C++",
+    "Ruby",
+  ];
+  const levels = ["Beginner", "Advanced", "Expert"];
+
+  const [skillList, setSkillList] = useState([{ language: "", level: "" }]);
+  const [submitClicked, setSubmitClicked] = useState(false);
+
+  const handleInputChange = (event, index) => {
+    const name = event.target.name;
+    const language = event.target.value;
+    const list = [...skillList];
+
+    list[index][name] = language;
+    setSkillList(list);
+  };
+
+  const handleAddClick = (event) => {
+    event.preventDefault();
+    setSkillList([...skillList, { language: "", level: "" }]);
+    setSubmitClicked(false);
+  };
+
+  const handleRemoveClick = (index) => {
+    const list = [...skillList];
+    list.splice(index, 1);
+    setSkillList(list);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    let allValid = true;
+
+    for (let i = 0; i < skillList.length; i++) {
+      allValid =
+        skillList[i].language && skillList[i].level && allValid ? true : false;
```

Comment:
I think the more standard way of writing this would be:

```
let allValid = true; 

for (let i = 0; i < skillList.length; i++) {
      if (!skillList[i].language || !skillList[i].level) {
            allValid = false;
            break;
     }
}
```

Would double check the logic and make sure it still works, but as soon as allValid becomes false, you can break